theme/scss/main.scss

```diff
@@ -1,14 +1,29 @@
 // CSS Reset for better defaults
 // https://github.com/sindresorhus/modern-normalize
 @import 'modern-normalize/modern-normalize.css';
+@import './app-bar.scss';
+@import './breakpoints.scss';
 
-html,
-body {
-  height: 100vh;
-  width: 100vw;
+// Variables
+:root {
+  --app-bar-height: 75px;
+  --main-col-width: 775px;
+  --side-col-width: 225px;
+  --col-margin: 25px;
```

Comment:
it looks like you updated the font sizes to scale from xs to sm but the design specified that *everything* should scale, so this should include things like the margins/paddings/etc.

Reply:
Good catch, I went ahead and added more `clamps()` to achieve the scaling affect. I didn't update all of the margins and paddings because I don't think everything has to adhere to that requirement, but I did add scaling to the important ones:

- App bar height
- Column margins
- App bar image width